.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Why do we need this file?

Reply:
I used this file a long time ago to try and launch some of the code in a VSCode debugger.  I abandoned that, so I don't need the file.

Reply:
I've deleted it.